src/olympia/constants/base.py

```diff
@@ -216,6 +216,7 @@
 MIN_TAG_LENGTH = 2
 MAX_CATEGORIES = 2
 VALID_CONTRIBUTION_DOMAINS = (
+    'flattr.com',
```

Comment:
Nit, but the list in in alphabetical order, so `flattr.com` would be after `donate.mozilla.org`